lms/static/js/certificates/views/certificate_bulk_whitelist.js

```diff
@@ -73,12 +74,91 @@
                 },
 
                 display_response: function(data_from_server) {
+                    var UserOnCertificateInvalidationList;
+
                     $('.bulk-exception-results').removeClass('hidden').empty();
 
+                    function generateDiv(group, heading, displayData) {
+                        // inner function generate div and display response messages.
+                        $('<div/>', {
```

Comment:
Ah, now I see why the XSS linter was giving you trouble -- the XSS linter is threshold-based, and you've added more linter failures to this file, such as in this block of code.

If you remove the `disable` comments you added, we can work with you to resolve the new XSS linter failures in this PR.

Reply:
@timmc-edx: I actually didn't add anything new here. I ran into a number of quality failures because I was the first person to touch this file in awhile. While fixing the quality issues I had to move around a bit of code and it appears that I added new pieces. I have had this work delayed a number of times due to quality changes I had to implement. I would love to fix this afterwards if this is acceptable.

Reply:
Oh! Sorry, I see now, this is moved code. Hmm.

I feel like something might be wrong with the linter, or the thresholds we've set. I'm going to do some investigation into that today.